docker/dev/docker-compose.yml

```diff
@@ -0,0 +1,60 @@
+version: "3.7"
+services:
+    opentogethertube-prod:
+        image: opentogethertube_dev
+        container_name: opentogethertube_dev
+        build:
+            context: ../../
+            dockerfile: docker/dev/Dockerfile
+        environment:
+            - YOUTUBE_API_KEY=API_KEY_GOES_HERE # optional put it in env/development.env
+            - REDIS_PASSWORD=""
+            - REDIS_DB=""
+        networks:
+            - opentogethertube_dev-net
+        ports:
+            - 3000:3000
+            - 3001:3001
+            - 8888:8888
+            - 6379:6379
+        links:
+            - redis
+        depends_on:
+            - redis
+            - postgres
+
+    redis:
+        container_name: opentogethertube_dev_redis
+        image: redis
+        environment: 
+            - REDIS_PASSWORD=""
+            - REDIS_DB=""
+        networks:
+            - opentogethertube_dev-net
+        volumes:
+            - dev_db_data:/data
+        expose:
+            - 6379
+
+    postgres:
```

Comment:
The dev environment doesn't need postgres since it uses sqlite3 instead. Production still needs it though